components/groups/utils.tsx

```
@@ -0,0 +1,154 @@
+  executeWithdrawal: () => void,
+  executeProposal: () => void,
+  setShowVotingPopup: (value: boolean) => void,
+  isSigning: boolean,
+  pollForData: boolean,
+  userVoteOption?: VoteOption | undefined
+): JSX.Element | undefined {
+  const isProposer = proposal.proposers.includes(address || '');
+  switch (proposalExecutorResultFromJSON(proposal.executor_result)) {
+    case ProposalExecutorResult.PROPOSAL_EXECUTOR_RESULT_NOT_RUN:
+      switch (proposalStatusFromJSON(proposal.status)) {
+        case ProposalStatus.PROPOSAL_STATUS_UNSPECIFIED:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)